iptables/table.go

```diff
@@ -263,6 +308,10 @@ func NewTable(
 
 		newCmd: newCmd,
 		sleep:  sleep,
+
+		gaugeNumChains:        gaugeNumChains.WithLabelValues(fmt.Sprintf("%d", ipVersion), name),
```

Comment:
Would `string(ipVersion)` work here too?  (As you used above)  I think that's nicer.